bindgen/src/bindings/cpp/templates/cpp_scaffolding.cpp

```diff
@@ -167,7 +176,7 @@ UNIFFI_EXPORT RustBuffer {{ ci.ffi_rustbuffer_alloc().name() }}(int32_t size, Ru
 }
 
 UNIFFI_EXPORT RustBuffer {{ ci.ffi_rustbuffer_from_bytes().name() }}(ForeignBytes bytes, RustCallStatus *out_status) {
-    out_status->code = 0;
+    out_status->code = CALL_STATUS_OK;
```

Comment:
Is `CALL_STATUS_OK` separated somehow from user lib namespace?

Reply:
This might have to be tackled better in separate MR, for now I've appended a `UNIFFI` prefix to these values to reduce the chances of clashing